index.bs

```diff
@@ -1066,13 +1087,114 @@ these steps:
 
 1. Let |top-level context| be |browsing context|'s [=top-level browsing context=].
 
-1. If |environment|'s [=environment settings object/origin=] is not [=same origin=] with |browsing context|'s [=top-level browsing context=]'s [=active document=]'s  [=/origin=],
+1. If |environment|'s [=environment settings object/origin=] is not [=same origin=] with |top-level context|'s [=active document=]'s  [=/origin=],
```

Comment:
Yep, it should probably. Although even better since some recent changes in html I can just use |environments|'s [=top-level origin=], simplifying things a bit.